Fix missing streams.h after libzerocash upgrade

I just tried to build `zc.v0.10.0.latest`, and it failed:

```
checking libzerocash/libzerocash/Zerocash.h usability... no
checking libzerocash/libzerocash/Zerocash.h presence... no
checking for libzerocash/libzerocash/Zerocash.h... no
configure: error: libzerocash headers missing
```

In `config.log` I found:

```
configure:26987: checking libzerocash/libzerocash/Zerocash.h usability
configure:26987: g++ -m64 -c -std=c++11 -pipe -O2 -O0 -g  -Wstack-protector -fstack-protector-all -fPIE -fvisibility=hidden -I/home/firexware/Data-1/Documents/FireXware/la/zerocashd/depends/x86_64-unknown-linux-gnu/include/libzerocash -I/home/firexware/Data-1/Documents/FireXware/la/zerocashd/depends/x86_64-unknown-linux-gnu/include/libsnark -DCURVE_ALT_BN128 -I/home/firexware/Data-1/Documents/FireXware/la/zerocashd/depends/x86_64-unknown-linux-gnu/include/  -DBOOST_SPIRIT_THREADSAFE -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS  -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 conftest.cpp >&5
In file included from /usr/include/stdio.h:27:0,
                 from conftest.cpp:61:
/usr/include/features.h:328:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
 #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
    ^
In file included from conftest.cpp:94:0:
/home/firexware/Data-1/Documents/FireXware/la/zerocashd/depends/x86_64-unknown-linux-gnu/include/libzerocash/libzerocash/Zerocash.h:17:21: fatal error: streams.h: No such file or directory
compilation terminated.
configure:26987: $? = 1
configure: failed program was:
```

--- a/configure.ac
+++ b/configure.ac
@@ -760,9 +760,11 @@ CPPFLAGS="-I$LIBZEROCASH_INCDIR $CPPFLAGS"
 
 # libzerocash depends on headers in ./src/, so for the following
 # AC_CHECK_HEADER, that has to be on the include path list.
+CPPFLAGS_TEMP="$CPPFLAGS"
 CPPFLAGS="-I ./src/ $CPPFLAGS"
 AC_CHECK_HEADER([libzerocash/libzerocash/Zerocash.h],,AC_MSG_ERROR(libzerocash headers missing))
 AC_CHECK_LIB([zerocash], [main],LIBZEROCASH_LIBS="-lzerocash -lsnark -lcryptopp -lgmp -lgmpxx -lzm -lboost_system-mt -lcrypto", [AC_MSG_ERROR(libzerocash missing)], [-lsnark -lcryptopp -lgmp -lgmpxx -lzm -lboost_system-mt -lcrypto])
+CPPFLAGS="$CPPFLAGS_TEMP"
 
 CFLAGS_TEMP="$CFLAGS"
 LIBS_TEMP="$LIBS"

